docs/developers/wallets.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_label: Wallets on Celestia
+description: An overview of wallets for users and developers on the Celestia network.
+---
+
+# Wallets on Celestia
+
+This category covers different types of wallets available for use on the
+Celestia network. Celestia's ecosystem offers a wide assortment of wallet solutions,
+providing users and developers options for storing and managing their TIA.
+
+From user-friendly interfaces to software binaries, this category will guide you
+through the offerings in the Celestia wallet space.
+
+## Wallets
+
+1. [Keplr wallet](../keplr)
+
+    Keplr is a widely recognized non-custodial wallet. The seamless integration with
+    Celestia ensures that users have a secure and user-friendly platform for managing
+    their assets. With the cross-chain capabilities of Keplr, users can manage, stake,
+    and swap tokens effortlessly.
+
+    1. [Ledger](../keplr) the Keplr guide will also show you how to use your Ledger
+    device to securely manage your account.
+
+2. [Cosmostation wallet](../cosmostation)
+
+    Renowned for its staking and defi capabilities, Cosmostation's integration with
+    Celestia pushes the boundaries of what's possible in the wallet ecosystem. Users
```

Comment:
would prefer if this was a bit more neutral so it doesn't sound like we're trying to entice readers into a specific option. Mainly just this part "cosmostation's integration with Celestia pushes the boundaries of what's possible in the wallet ecosystem"

Reply:
have applied this suggestion and removed descriptions for now as this is just an overview page to direct folks